### PR TITLE
separate docs building

### PR DESCRIPTION
**Pull request by levitte**
Make it possible to build the docs subdirectory entirely separately

### CODE REVIEW

**Review by t8m — APPROVED** [review]

**Comment by levitte** [comment]
A second reviewer needed

File: Makefile [hunk 1]
```diff
@@ -75,21 +70,30 @@ SIMPLE = newsflash.inc \
...
+	     docs/mansidebar.html
 
 .SUFFIXES: .md .html
 
 .md.html:
 	@rm -f $@
 	./bin/md-to-html5 $<
 
-all: suball manmaster mancross sitemap akamai-purge
+all: suball docs sitemap akamai-purge
```
Comment (mattcaswell) [inline Makefile]: This seems to actually change the "all" target. Previously only `manmaster` and `mancross` was included. Now its `manpages` and `mancross`. So `manpages` instead of `manmaster` which seems to encompass more. Is that change intentional?

Comment (levitte) [reply Makefile]: Ah, good point. Will fix

Comment (levitte) [reply Makefile]: Fixed

**Review by mattcaswell — APPROVED** [review]

**Comment by levitte** [comment]
Merged

ef48d3f5c047651da3db9323a474f537f3c104d3 Make it possible to build the docs subdirectory entirely separately
